recipes/gifrop/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "gifrop" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "7059c916b2deec451a97037936bc4905c44211d12a87d9db3e8724a9797672f0" %}
+{% set user = "jtrachsel" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/{{ user }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+ host:
```

Comment:
Please remove the `host:` section, it isn't needed for this recipe.

Reply:
Thanks for looking at this so quickly.  I was unable to get this to build without at least a host entry for abricate.  I have two custom databases that need to be set up. I set up these databases in the build script by using the output of the `abricate --list` command to put the files in the correct location.  Is there a better way of doing this?

Reply:
That's totally fine.